src/cpp/session/modules/SessionDataPreview.R

```diff
@@ -0,0 +1,66 @@
+#
+# SessionDataPreview.R
+#
+# Copyright (C) 2009-18 by RStudio, Inc.
+#
+# Unless you have received this program directly from RStudio pursuant
+# to the terms of a commercial license agreement with RStudio, then
+# this program is licensed to you under the terms of version 3 of the
+# GNU Affero General Public License. This program is distributed WITHOUT
+# ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+# MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+# AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+#
+#
+
+.rs.addFunction("previewDataFrame", function(data, script)
+{
+   max <- 100
+
+   columns <- list()
```

Comment:
should this share an implementation with https://github.com/rstudio/rstudio/blob/560aeb25f6634b3187082280e192d595ebc047be/src/cpp/session/modules/SessionDataImportV2.R#L767-L774?

Reply:
Can do.